src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -271,38 +208,5 @@ private static Request ToRequest(FileConfig fileConfig, string path)
             }
             return request;
         }
-
-        private static void AddRange(IList<string> cmd, string[] values)
```

Comment:
These methods were only used by the nested `Capture` class, so I've moved them to that class.